auth-web/src/components/auth/common/Product.vue

```diff
@@ -8,6 +8,7 @@
         class="product-card py-8 px-5 mb-4 elevation-1"
         :class="[ {'processing-card' : isSelected}]"
         :data-test="`div-product-${productDetails.code}`"
+        v-if="!productDetails.hidden"
```

Comment:
not sure how other people feel about these, but I find it really helpful to have v-if/for's at the beginning

Reply:
sounds good I'll move it back